deployment/CP05MOAS-GL388_Deploy.csv

```diff
@@ -1,61 +1,67 @@
+TS20191212,,TS20200301,,CP05MOAS-GL388-02-FLORTM000,9,1,2019-12-12T22:47:17,2020-03-02T06:49:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-FLORTM-03185,39.914233,-70.665167,,1000,1000,SS-1,,
+TS20191212,,TS20200301,,CP05MOAS-GL388-03-CTDGVM000,9,1,2019-12-12T22:47:17,2020-03-02T06:49:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-00073,39.914233,-70.665167,,1000,1000,SS-1,,
+TS20191212,,TS20200301,,CP05MOAS-GL388-04-DOSTAM000,9,1,2019-12-12T22:47:17,2020-03-02T06:49:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00323,39.914233,-70.665167,,1000,1000,SS-1,,
+TS20191212,,TS20200301,,CP05MOAS-GL388-05-PARADM000,9,1,2019-12-12T22:47:17,2020-03-02T06:49:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50244,39.914233,-70.665167,,1000,1000,SS-1,,
+TS20200613,,TS20200819,,CP05MOAS-GL388-00-ENG000000,10,1,2020-06-13T21:49:01,2020-08-19T22:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGCON-GLDRCP-00388,39.92978,-70.36552,,1000,1000,SS-2,,
+TS20200613,,TS20200819,,CP05MOAS-GL388-01-ADCPAM000,10,1,2020-06-13T21:49:01,2020-08-19T22:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-ADCPAM-49016,39.92978,-70.36552,,1000,1000,SS-2,,
+TS20200613,,TS20200819,,CP05MOAS-GL388-02-FLORTM000,10,1,2020-06-13T21:49:01,2020-08-19T22:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-FLORTM-03185,39.92978,-70.36552,,1000,1000,SS-2,,
+TS20200613,,TS20200819,,CP05MOAS-GL388-03-CTDGVM000,10,1,2020-06-13T21:49:01,2020-08-19T22:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-00073,39.92978,-70.36552,,1000,1000,SS-2,,
+TS20200613,,TS20200819,,CP05MOAS-GL388-04-DOSTAM000,10,1,2020-06-13T21:49:01,2020-08-19T22:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00323,39.92978,-70.36552,,1000,1000,SS-2,,
+TS20200613,,TS20200819,,CP05MOAS-GL388-05-PARADM000,10,1,2020-06-13T21:49:01,2020-08-19T22:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50244,39.92978,-70.36552,,1000,1000,SS-2,,
+TS20210517,,,,CP05MOAS-GL388-00-ENG000000,11,1,2021-05-17T20:25:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGCON-GLDRCP-00388,39.81837,-70.3787,,1000,1000,FZ-1,,
+TS20210517,,,,CP05MOAS-GL388-01-ADCPAM000,11,1,2021-05-17T20:25:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-ADCPAM-55979,39.81837,-70.3787,,1000,1000,FZ-1,,
+TS20210517,,,,CP05MOAS-GL388-02-FLORTM000,11,1,2021-05-17T20:25:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-FLORTM-03884,39.81837,-70.3787,,1000,1000,FZ-1,,
+TS20210517,,,,CP05MOAS-GL388-03-CTDGVM000,11,1,2021-05-17T20:25:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-09055,39.81837,-70.3787,,1000,1000,FZ-1,,
+TS20210517,,,,CP05MOAS-GL388-04-DOSTAM000,11,1,2021-05-17T20:25:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00170,39.81837,-70.3787,,1000,1000,FZ-1,,
+TS20210517,,,,CP05MOAS-GL388-05-PARADM000,11,1,2021-05-17T20:25:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50324,39.81837,-70.3787,,1000,1000,FZ-1,,
```

Comment:
QLR deploy table lists EB for assigned line (but the main part of the QLR lists FZ-1.  Assume FZ-1 is correct since this is a 1000 m glider.  The rest looks ok.

Reply:
388 is on the FZ-1 line. Deploy CSV correct. fixed QLR.

Reply:
roger that!  looks good then.

Reply:
Looks good.